packages/dm-core/src/components/EntityView.tsx

```diff
@@ -28,13 +37,29 @@ export const EntityView = (props: IEntityView): React.ReactElement => {
     dimensions
   )
 
+  // Refresh Button stuff
+  const [reloadCounter, setReloadCounter] = useState(0)
+  const [mouseHoverOverRefreshButton, setMouseHoverOverRefreshButton] =
+    useState(false)
+  const [mouseCloseToRefreshButton, setMouseCloseToRefreshButton] =
+    useState(false)
+  const reloadComponent = () => setReloadCounter(reloadCounter + 1)
```

Comment:
Not a big fan of these functions that only call another function. IMO it's better to call "setReloadCounter" directly.
Helps with readability

Reply:
alright